etc/profile-a-l/chafa.profile

```diff
@@ -0,0 +1,56 @@
+# Firejail profile for chafa
+# Description: chafa is an image viewer.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include chafa.local
+# Persistent global definitions
+include globals.local
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+# Users may want to view images in ${HOME}
+#include disable-xdg.inc
+
+# Users may want to view images in ${HOME}
+#include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+# Users may want to view images in /usr/share
+#include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+machine-id
+net none
+nodvd
+nogroups
+noinput
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix
```

Comment:
Note for me: Only if it has x11 access.

Reply:
It does not need x11 access. So I guess we can remove these lines and add `x11 none` after `tracelog`?

Reply:
Yes.

- Add `x11 none`
- Add `blacklist ${RUNUSER}/wayland-?` or `blacklist ${RUNUSER}`
- Remove `protocol unix`
- Replace `seccomp` with `seccomp socket`